tools/Assembly/GFF/build_assembly_gff.py

```diff
@@ -213,20 +238,27 @@ def build_gff(annotations, faa):
         '-o', dest='out', help='Ouput GFF file name', required=True)
     args = parser.parse_args()
 
-    with open(args.out, 'w', buffering=1) as out_handle:
+    annotations = {}
+    load_annotation(args.egg, EggResult, annotations)
+    load_annotation(args.interpro, InterProResult, annotations)
 
-        print('##gff-version 3', file=out_handle)
-
-        annotations = {}
-        load_annotation(args.egg, EggResult, annotations)
-        load_annotation(args.interpro, InterProResult, annotations)
+    if len(annotations) < 1:
+        print("No annotations loaded, aborting")
+        sys.exit(1)
 
+    records = 0
+    with open(args.out, 'w', buffering=1) as out_handle:
```

Comment:
I'm curiuos, why `buffering=1`?

Reply:
It was there before me, I guess because it is writing text files line by line

Reply:
You are right, apologies for the confusion he.